utilities/constants.py

```diff
@@ -112,6 +112,7 @@ class KubernetesAnnotations:
     INSTANCE: str = f"{Resource.ApiGroup.APP_KUBERNETES_IO}/instance"
     PART_OF: str = f"{Resource.ApiGroup.APP_KUBERNETES_IO}/part-of"
     CREATED_BY: str = f"{Resource.ApiGroup.APP_KUBERNETES_IO}/created-by"
+    DEPLOYMENT_MODE: str = "serving.kserve.io/deploymentMode"
```

Comment:
Wouldn't it be better to put this under a different class, e.g. `KserveAnnotations`, or rename the class to be more general?

Reply:
split into subclasses